sys/check_meson_subproject.py

```diff
@@ -12,6 +13,9 @@
 import os
 import sys
 
+def eprint(*args, **kwargs):
+    print(*args, file=sys.stderr, **kwargs)
```

Comment:
what is this?

Reply:
I forgot to remove it. sorry. i used it for debugging